public class Main5 {
	public static void main(String[] args) {
		try {
			int i=10;
		}
		catch(Exception e) 
		{
			System.out.println("Got the Exception" +e);
		}
	}
}
